templates/patterns/list_item/pattern-list-item--variant-navigation.html.twig

```diff
@@ -4,10 +4,8 @@
  * Default implementation for the navigation list item component.
  */
 #}
-{% include '@oe_theme/compositions/ec-component-content-item/content-item.html.twig' with {
-  title: {
-    label: title,
-    path: url
-  },
-  description: detail|smart_trim(length)
+{% include '@oe_theme/patterns/list_item/pattern-list-item.html.twig' with {
+  title: title,
+  url: url,
+  detail: detail|smart_trim(length),
```

Comment:
We apply the same filter in the base pattern so I think we can skip it here.

Reply:
Removed